practice-app/website/api/discussion_forum.py

```diff
@@ -34,18 +34,26 @@ def bad_word_check(body):
 @forum.route('/forum_get/', methods=["GET"])
 def forum_get():
 
+    """
+    file: ./doc/forum_get_GET.yml

```

Comment:
I think you can rename these files as `forum_POST.yml` and `forum_GET.yml` for simplicity.